app/models/heritage.rb

```diff
@@ -255,6 +255,7 @@ def image_path
   end
 
   def deploy!(without_before_deploy: false, description: "")
+    validate_ssm_parameters
```

Comment:
Add an exclamation mark here since its a method that does not return anything.

```suggestion
    validate_ssm_parameters!
```